protocol/diff_test.sh

```diff
@@ -16,9 +16,18 @@ make
 # This build process is idempotent. We expect there to be no changes after
 # re-running the protocol generation, because any changes should have been
 # checked in.
+
 if ! git diff --exit-code ./ ../go/ ../shared/; then
```

Comment:
Unless you want the cases to be separated, in which case the current snippet is fine